app/model/commands/PublishAtomCommand.scala

```diff
@@ -99,9 +99,19 @@ case class PublishAtomCommand(id: String, override val stores: DataStores, youTu
   }
 
   private def updateYouTube(previewAtom: MediaAtom, asset: Asset): Future[MediaAtom] = {
-    updateYoutubeMetadata(previewAtom, asset)
-    updateYoutubeThumbnail(previewAtom, asset)
-    createOrUpdateYoutubeClaim(previewAtom, asset)
+    previewAtom.channelId match {
+      case Some(channel) if youTube.allowedChannels.contains(channel) =>
```

Comment:
As the main channel isn't in the allowed list, we're now able to add an asset with the YT url but will never update YT metadata.

We store a list of channels linked to the cms [in the cache](https://github.com/guardian/media-atom-maker/blob/master/app/controllers/Youtube.scala#L21). We could see if the video channel is in that list, and then treat it as a first party video? Might need some ux attention though. Lets ship this and see.